workspaces/rbac/plugins/rbac-backend/README.md

```diff
@@ -241,3 +241,31 @@ More information about group hierarchy can be found in the doc: [Group hierarchy
 ### Optional RBAC provider module support
 
 We also include the ability to create and load in RBAC backend plugin modules that can be used to make connections to third part access management tools. For more information, consult the [RBAC Providers documentation](./docs/providers.md).
+
+### Configure default permissions
+
+By default, users who don't have any permissions assigned won't be able to access any resources. However, you can configure default permissions that apply to such users, ensuring a minimum level of access for everyone.
+
+#### Detailed format (multiple specific permissions):
+
+```YAML
+permission:
+  enabled: true
+  rbac:
+    defaultUserAccess:
+      enabled: true # Enable/disable the default access feature
+      defaultPermissions:
+        - permission: catalog-entity # Resource type
+          policy: read
+          effect: allow
+        - permission: scaffolder.template.step.read # Permission name
+          policy: read
+          effect: allow
```

Comment:
Do you mind adding these new configuration values to the [config.d.ts](https://github.com/backstage/community-plugins/blob/main/workspaces/rbac/plugins/rbac-backend/config.d.ts) file, please?

Reply:
@PatAKnight done 👍